### PR TITLE
{tools}[intel/2017a] googletest v1.8.0

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/g/googletest/googletest-1.8.0-intel-2017a.eb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+easyblock = 'CMakeMake'
+
+name = 'googletest'
+version = '1.8.0'
+
+homepage = 'https://github.com/google/googletest'
+description = "Google's C++ test framework"
+
+toolchain = {'name': 'intel', 'version': '2017a'}
```
Comment (wpoely86) [inline easybuild/easyconfigs/g/googletest/googletest-1.8.0-intel-2017a.eb]: `GCCcore`

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2478.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/bdbb382c1f43c17c055eb728c2fa41d0 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2130.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/cd274197bf5ba7184268251284163eb9 for a full test report.

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**